schema.json

```
@@ -1,329 +1,423 @@
+      "example": "Accélérateur décarbonation",
+      "type": "string",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "promesse",
+      "description": "La promesse a pour objectif de :\n- résumer le dispositif de manière claire & simple en quelques mots\n- distinguer les offres les unes des autres.\n\nFormulation :\n- 6 à 15 mots.\n- La promesse commence par un verbe à l'impératif\n- rédigée sous l'angle du bénéfice utilisateur, s'adressant à lui par le pluriel de politesse \"vous\" \"votre entreprise\" \n- pas de point à la fin de la promesse (sauf point d'exclamation)",
+      "example": "Développez votre stratégie bas carbone et mettez en œuvre votre plan de transition",
+      "type": "string",
+      "constraints": {
+        "required": true,
+        "maxLength": 180
+      }
+    },
+    {
+      "name": "description_courte",
+      "description": "Description courte de l'objectif du dispositif. Qu'est-ce que l'entreprise gagne à bénéficier de cette aide ?\n\nFormulation :\n- 40 à 60 mots\n- rédigée sous l'angle du bénéfice utilisateur, s'adressant à lui par le pluriel de politesse \"vous\" \"votre entreprise\"\n- La description courte est ponctuée par un point.",
+      "example": "L’ADEME soutient financièrement les entreprises qui souhaitent questionner leur modèle économique pour identifier un nouveau modèle compatible avec une trajectoire bas carbone. Vous serez accompagné avec la méthode ACT Pas à Pas (Assessing Low Carbon Transition), une méthode structurante, progressive et sur le long terme qui aide les entreprises à réduire significativement leurs émissions de gaz à effet de serre.",
+      "type": "string",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "description_longue",
+      "description": "Description longue du dispositif - facultatif, texte enrichi en markdown. Elle peut être ajoutée si on juge que l'utilisateur ne dispose pas des informations suffisantes avec les seules autres informations. Elle peut être ajoutée par exemple :\n- pour définir un terme technique\n- pour détailler une loi à laquelle un dispositif fait référence\n- pour détailler les projets concernés par un financement\n\nFormulation :\n- 50 à 200 mots",
+      "example": "L'étude ACT Pas à Pas se déroule en 12 à 18 mois selon les entreprises. Les grandes étapes de l'étude sont les suivantes :\n- Analyser la maturité initiale de l'entreprise sur des bases carbone.\nExposer les enjeux et réaliser une analyse stratégique de l’entreprise dans un contexte de transition bas carbone.\n- Déterminer une vision d’entreprise et concevoir la feuille de route du plan de transition à court, moyen et long terme.\n- Elaborer des objectifs de performance carbone et définir une stratégie.\n- Mettre en oeuvre des actions de décarbonation.",
+      "type": "string"
+    },
+    {
+      "name": "programme_aides",
```

Il peut y avoir plusieurs programmes, mais ça concerne toujours une seule aide (si j'ai bien compris ?)
```suggestion
      "name": "programmes_aide",
```

---

:+1: le champ existait déjà et j'ai tendance à trop garder de legacy lorsque c'est possible ... alors même que la PR propose une très grosse refonte du schéma...